### DISCUSSION TITLE
Welcome to SE-Project Discussions!

### DISCUSSION THREAD

**Discussion by vinay7244** [vinay7244/SE-Project#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vinay7244** [comment]
hi

**Reply by sathyajee** [reply]
hello

**Comment by sathyajee** [comment]
what is this
